src/components/words/Words.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Words({ words, color }) {
+  const wordsList = words.map(word => {
+    return (
+      <li key={word}>{word}</li>
+    );
+  });
+  return (
+    <ul style={{ color: color }}>
+      {wordsList}
```

Comment:
```suggestion
      {wordsList.length > 0 ? wordsList : 'No entries found' }
```
Little bit of error handling :)